src/theme/styled.d.ts

```diff
@@ -284,6 +284,27 @@ declare module 'styled-components' {
         bgd: string;
       }
     }
+
+    messageAttachment: {
+      size: {
+        fontColor: string;
+        bgd: string;
+        letterSpacing: string;
+        lineHight: string;
+        fontSize: string;
```

Comment:
If these values match the BaseStyles values we don't need them here.

Reply:
Yeah! This do not! it's very specific.